static/resources/harness.js

```diff
@@ -693,6 +693,43 @@
     }
   }
 
+  function loadHighlightJs(callback) {
+    try {
+      // Load dark (main) style
+      var darkStyle = document.createElement('link');
+      darkStyle.rel = 'stylesheet';
+      darkStyle.href =
+        '//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.3.1/styles/stackoverflow-dark.min.css';
```

Comment:
Can we put everything in resources/ to avoid this?

Reply:
I wanted to do that, but this turns out this was a bigger hurdle than I originally thought.  We'll have to build our own bundle, which while I'd love to do that (since we can pick and choose syntax languages to minify our bundle to our perfect needs), I figured it would be better to use the pre-built bundle for now!